openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -118,6 +118,10 @@ Both the Wallet and the Credential Issuer:
 
 Both Wallet initiated and Issuer initiated issuance is supported.
 
+## Issuer Metadata
+
+Support for signed Issuer Metadata to allow for Issuer authentication via the `signed_metadata` parameter in the Issuer Metadata is RECOMMENDED. When required by ecosystem policy, signed Issuer Metadata MUST be supported by both the Wallet and the Issuer. Key resolution to validate the signed Issuer Metadata via the `x5c` header parameter in the JWT MUST be supported for signed Issuer Metadata.
```

Comment:
```suggestion
To support signed Issuer Metadata to allow for Issuer authentication, it is
RECOMMENDED to use the `signed_metadata` parameter in the Issuer Metadata.
When required by ecosystem policy, signed Issuer Metadata MUST be supported by
both the Wallet and the Issuer. Key resolution to validate the signed Issuer Metadata
MUST be supported using the `x5c` header parameter in the JWT.
```
I would like to be more explicit.

+ can we please put reference to `x5c` parameter?